tests/testthat/test-dm.R

```diff
@@ -370,10 +330,14 @@ test_that("dm_get_src() works", {
     class = "is_not_dm"
   )
 
-  walk2(
-    dm_for_filter_src,
-    active_srcs_class,
-    ~ expect_true(inherits(dm_get_src(.x), .y))
+  expect_identical(
+    class(dm_get_src(dm_for_filter)),
+    c("src_local", "src")
```

Comment:
```suggestion
    class(my_test_src)
```

with an appropriate `my_test_src` .

Perhaps even `expect_dm_src(dm_for_filter, my_test_src)` ?

Reply:
Ok, I'll use a dummy local `my_test_src` for now